Add meta descriptions (#4003)
wazuh/wazuh-documentation: source/docker/container-usage.rst

.. Copyright (C) 2021 Wazuh, Inc.

.. meta::

>   :description: Check the tasks that help you benefit the most from the installation of Wazuh after the installation of the Wazuh-Docker. 

Reviewer: ```suggestion
  :description: Check the tasks that help you benefit the cvgbdfgfrom the installation of Wazuh after the installation of the Wazuh-Docker. 
```
